TM1py/Utils/Utils.py

```diff
@@ -74,6 +74,32 @@ def get_all_servers_from_adminhost(adminhost='localhost') -> List:
         servers.append(server)
     return servers
 
+def create_server_on_adminhost(adminhost='localhost', server_as_dict={}):
+    from TM1py.Objects import Server
+    """  Create new TM1 instance on Adminhost
+    :param adminhost: IP or DNS Alias of the adminhost
+    :param server_as_dict: 
+            server_as_dict = {
+                "Name":"MyModel1",
+                "IPAddress":"172.20.10.10",
+                "IPv6Address":None,
+                "PortNumber":12345,
+                "UsingSSL": True,
+                "ClientMessagePortNumber":61098,
+                "HTTPPortNumber":12999,
+                "ClientExportSSLSvrCert":True,
+                "ClientExportSSLSvrKeyID":"whateverExportSSLSvrKeyID",
+                "AcceptingClients":True }
+    :return: instance of TM1py.Server
+    """
+
+    conn = http_client.HTTPConnection(adminhost, 5895)
+    request = '/api/v1/Servers'
+    conn.request('POST', request, body=json.dumps(server_as_dict), headers={'Content-Type':'application/json'})
+    response = conn.getresponse().read().decode('utf-8')
+    response_as_dict = json.loads(response)
+
+    return Server(response_as_dict)
```

Comment:
I think we should enhance the existing Server class. 
The class should support all fields that are returned in the response dictionary.

Reply:
I've added all returned properties. I haven't been able to verify all possible outcomes, i.e. local v hosted.